flan/src/routes/chat/+page.svelte

```diff
@@ -155,6 +154,27 @@
 		inputFile = null;
 	}
 
+	function onConnClose(): void {
+		console.log('Connection closed');
+		const toastMessage: ToastSettings = {
+			message: 'Session disconnected 😿',
```

Comment:
Does this also have no close button on the toast, please include image as reference

Reply:
It does have a close button, just didn't have the autohide so it stays on the screen. Shows up on both peers' screens when session is disconnected. Let me know if functionality should be that it only shows up for the person still in the chat. I got QA where users were under the impression that they could re-join the chat so it might be beneficial to have the toast disconnect message appear for both peers.

Reply:
Yep yep that's intended, I meant as in I don't think the user should be able to close that popup or it should keep reappearing because otherwise we need some other way to signify that the chat is in a disconnected state

Reply:
Here's an image for context
<img width="1234" alt="image" src="https://github.com/ProPablo/nakl/assets/45290942/5e50d45b-83c6-4fc4-829a-24884e3154c4">

Reply:
Ok just tested, this also runs everytime we go back to home too, probably not ideal, dunno a good way to solve as of yet tho

Reply:
oop, might need to check the current route and conditionally render the toast based on that?

Reply:
Wouldn't you want it to show in both instances? Because when navigating home you are essentially disconnecting from your current session right? 

Or do you mean that the prompt shows up regardless of when you navigate home from the website?

Reply:
Actually there's an on dismount similar to react which is return in onmount, so.e logic needs to be done there to fix this I imagine

Reply:
Addressed. With the toast now being auto-hide as well
<img width="395" alt="image" src="https://github.com/ProPablo/nakl/assets/45290942/2a43704e-e7ab-4f91-968b-e8b4d6065021">